### ISSUE TITLE
ENV Variables in request highlighting 

### ISSUE THREAD

**Issue by NicOfNoomers** [hoppscotch/hoppscotch#1029]
**Is your feature request related to a problem? Please describe.**
Trying to use environment variables is pretty difficult, and we never know if a variable is successfully attached to the request

**Describe the solution you'd like**
If there was some kind of checking with underline or changing text color it would be much easier to identify wrong env. variables

**Describe alternatives you've considered**
Insomnia Rest does smth like this, as shown in the screenshot

**Additional context**
Attached Screenshots
![image](https://user-images.githubusercontent.com/19762533/88581142-672cac80-d055-11ea-96bf-0ad7eccef99f.png)

**Comment by sukheja-varun** [comment]
I totally agree, in Insomnia it hardly took any time to figure out how env variables work but in postwoman, it is a bit difficult. 
I have created an env with few variables but still not able to figure out how to make any env active.

**Comment by liyasthomas** [comment]
Got it. Since `envs` are used inline inside an input box (`URL` field), it's quite hard to list / highlight registered envs from input. I tried to parse `URL` input with some regex to show registered envs in  a tooltip but had no luck with it. Will try another method. Let me know if there's any UI wise implementation that could tackle this. May be like auto-complete for envs?

**Comment by liyasthomas** [comment]
That's great input. Visually, this is something I'd in mind. Let me dig deep into this approach. Will ping you in need of any help ;)

**Comment by AndrewBastin** [comment]
Umm, I am actually working on the implementation of this. Along with a overhaul of the URL parsing system. Just that all new feature work is paused for a while to work on tests and improving the stability and reliability until the Teams feature has rolled out.

**Comment by AndrewBastin** [comment]
Update: Environment variable highlighting is now implemented in the URL field as an Experiment. You can enable the experiment in the settings page. It is not stable yet, but it is on track to be stable soon. 

Closing this issue meanwhile.

**Comment by yoerriwalstra** [comment]
I feel a bit dumb for having to ask this, but what is the syntax for using environment variables in your URLs?

I have an environment with variable `base_url` in it, but I can't get my requests to work with it... I couldn't find any examples showcasing this feature, so I had to guess how it should work. I read the discussion in #147 which suggested that Hoppscotch would, like Postman, [support nunchucks syntax for environment variables](https://github.com/hoppscotch/hoppscotch/issues/147#issuecomment-542352975). I've tried different variations, but without success.

Example:
```
GET {{ environment.base_url }}/companies
GET {{ variables.base_url }}/companies
GET {{ base_url }}/companies
```
